roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -13,8 +13,16 @@
 
 
 class OpenShiftCheckException(Exception):
-    """Raised when a check cannot proceed."""
-    pass
+    """Raised when a check encounters a failure condition."""
+
+    def __init__(self, name, msg=None):
+        # msg is for the message the user will see when this is raised.
+        # name is for test code to identify the error without looking at msg text.
+        if msg is None:  # for parameter backward compatibility
```

Comment:
Of course, but I like to have the concise identifier first and then the blah blah blah when I'm looking at the code raising an exception. Selfish of me? And for once I thought it would be nice not to force a total transformation to the new method signature.